### PR TITLE
Portfolio: mobile version

### PR DESCRIPTION
**Pull request by joel-kalema**
- [x] Follow the list of best practices for HTML & CSS
- [x] Now I will build the middle section of the mobile website. We included a screenshot of Template 1 below for reference, but 
       I should stick to the template that I chose. These sections of the mobile website should include:
           -"Works" section
           -the "About myself" section (placed after the projects and before the contact form)
- [x] In order to lay out the elements in the "Works" section I should use CSS Grid and Flexbox combined (it is mandatory that 
       you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to layout an individual card.
- [x] I must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma
- [x] Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,16 +68,113 @@ <h5>Skills</h5>
...
+      </section>
+
+      <section class="works projects">
+        <span class="works1"></span>
+        <span class="works2"></span>
+        <div class="down">
+          <i class="fa fa-light fa-arrow-down"></i><span>Work</span>
+        </div>
+        <h3>MY RECENT</h3>
+        <h3>WORK</h3>
```
Comment (MrBrN197) [inline index.html:87]: - [ ] Hi, this `.works.projects` section should use `display: grid` as per the requirements. please use `grid` in order to become accustomed to how it works.

File: index.html [hunk 2]
```diff
@@ -70,16 +68,113 @@ <h5>Skills</h5>
...
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <div class="langage-used">
+            <ul>
+              <li>Css</li>
+              <li>Html</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
```
Comment (MrBrN197) [inline index.html]: - [ ] Please notice that in the technologies section on the reference. each technology tag is separated with a small thin vertical bar. please add this in order to practice matching a reference design as best as you can. 

|Your Design | Reference Design|
-|-
|image|image|

File: index.html [hunk 3]
```diff
@@ -70,16 +68,113 @@ <h5>Skills</h5>
...
+          <button type="button">See Project</button>
```
Comment (MrBrN197) [inline index.html:107]: - [ ] Please notice that the "See project" button when hovered doesn't have the correct color. please use the correct color suggested in the Figma. that color should be `#5E42D7` when hovering over the button

|Your Design|Reference Design|
-|-
|image|image|

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @joel-kalema  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (joel-kalema) [reply index.html:107]: about the button, I gave it a hover color but it has a normal color

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html:87]: 👍🏽 This is well implemented.

File: index.html (hunk 2 above)
Comment (Whoistolu) [reply index.html]: 👍🏽 Also. well implemented.

**Review by Whoistolu — APPROVED** [review]
### Hi @joel-kalema,

Thank you for implementing the changes recommended by the previous reviewer.
Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**